smartcontractkit/external-adapters-js
# [PDI-178] Invalid address handling for eth-beacon

## Closes [PDI-178](https://smartcontract-it.atlassian.net/browse/PDI-178)

## Description

Updated the `eth-beacon` implementation to append results with addresses filtered out due to their status or being invalid with a 0 balance. This ensures that the results have some values in them so they do not fail at the reduce step but still yield the same results as if those addresses were filtered out.

## Changes

- Added logic to append addresses missing from the results that were in the original request with a balance of 0

## Steps to Test

1. Start the `eth-beacon` adapter
2. Send a request with an invalid address
3. Notice results contains all of the addresses from the request and the invalid address shows a balance of 0

## Quality Assurance
